script/Arcueid/Day 9/09_00_ARC09_2_2.txt

```diff
@@ -108,7 +108,7 @@ Kohaku-san, you can leave the first-aid kit behind. You're coming right back, af
 [sha:aa2900d4d568fa65cc335602296c33d2721aa10e]{
 -- Page 79, Offset 21511.
 -- 「いえ、こちらはお気持ちを鎮める<薬|もの>ばかりなので、打ち身専用のものと持ち替えて参ります。
-"Ah, no this kit is full of medicines to calm the mind,
+"Ah no, this kit is full of medicines to calm the mind,
```

Comment:
Doesn't feel like a natural pause config to me. Either just `Ah, no this kit` or `Ah, no, this kit` better match how one might say it I think.